features/naam/naam.feature

```diff
@@ -49,86 +49,176 @@
       | code         | M      |
       | omschrijving | man    |
 
+  Scenario: naam van de bewoner met diakrieten wordt geleverd bij het raadplegen van bewoning met periode
```

Comment:
dit is een bewoner zonder diacrieten, dus ik denk dat de scenariotitel niet klopt